cardano-testnet/test/cardano-testnet-test/cardano-testnet-test.hs

```diff
@@ -105,4 +108,6 @@ main = do
   hSetEncoding stdout utf8
   args <- E.getArgs
 
-  E.withArgs args $ tests >>= T.defaultMainWithIngredients ingredients
+  let opts = T.singleOption $ T.NumThreads 1
```

Comment:
Can we have environment variable to override this? It's still useful to me to be able to run tests in parallel locally.

Reply:
You should be able to do this:

```
--test-options '--num-threads 8'
```

Reply:
The default is:

```
  defaultValue = NumThreads numCapabilities
```

Whatever that works out to be.